package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.4",
   "description": "",
   "scripts": {
-    "start": "DEBUG=ERROR-*,WARN-*,INFO-* node build/src/bot",
-    "start:env": "DEBUG=ERROR-*,WARN-*,INFO-* node -r dotenv/config build/src/bot",
-    "dev": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config  -r tsconfig-paths/register src/bot.ts",
+    "start": "DEBUG=ERROR-*,WARN-*,INFO-* NTBA_FIX_319=yes node build/src/bot",
```

Comment:
What is that even NTBA_FIX_319 ? Do we need to use a 3rd-party promise library with cancellable promises? Like bluebird?

Reply:
I think it's just the opposite. I don't need it, nor the warning:
https://github.com/yagop/node-telegram-bot-api/commit/f4f015a6219fc8efdb521dad72fb860361235de8#diff-625674fbfd357ad2c4a72a161ae8ad2dR53